modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
Aha, ok.. so this is a timeout for deciding overload or ok? I thought it was a timeout to check for completion of a sidejob.